src/gpu/intel/jit/dsl/dsl.hpp

```diff
@@ -34,6 +34,7 @@ int grf_size();
 int min_align_2d();
 int min_pitch_2d();
 
+using type_t = dnnl::impl::gpu::intel::jit::type_t;
```

Comment:
nit: this seems like it will need declared in `dsl/decl.hpp` due to the intended namespace changes when upstreaming.